geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/netty/ExecutionHandlerContext.java

```diff
@@ -226,15 +227,9 @@ private RedisResponse getExceptionResponse(ChannelHandlerContext ctx, Throwable
       return null;
     }
 
-    if (cause instanceof FunctionException
-        && !(cause instanceof FunctionInvocationTargetException)) {
-      Throwable th = getInitialCause((FunctionException) cause);
```

Comment:
Is it no longer possible to hit a `FunctionException` here?  I forget why this was added....

Reply:
Since we've removed all function execution from command calling we no longer need this.